### PR TITLE
updating the read-me file

### LINKED ISSUE #33: [CHORE] Update the READ.me
**Is your feature request related to a problem? Please describe.**
We need improved documentation for the read.me

### PR DESCRIPTION
**Pull request by swimking**
## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by LadPaule — COMMENTED** [review]
is it okay to add the link to the netlify provisional space
https://farmrail-staging.netlify.app/

File: README.md [hunk 1]
```diff
@@ -3,3 +3,29 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/b13777f5d669412cb02635a66b5de4a8)](https://app.codacy.com/gh/BuildForSDG/farmrail?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/farmrail&utm_campaign=Badge_Grade_Dashboard)
 
 Build for SDG project repo
```
Comment (amkayondo) [inline README.md:5]: Place this under the Title

File: README.md [hunk 2]
```diff
@@ -3,3 +3,29 @@
...
+## Project Name: Farm Rail
+
+PRODUCT DESCRIPTION
```
Comment (amkayondo) [inline README.md:8]: bold this

File: README.md [hunk 3]
```diff
@@ -3,3 +3,29 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/b13777f5d669412cb02635a66b5de4a8)](https://app.codacy.com/gh/BuildForSDG/farmrail?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/farmrail&utm_campaign=Badge_Grade_Dashboard)
 
 Build for SDG project repo
+## Project Name: Farm Rail
```
Comment (amkayondo) [inline README.md:6]: make it bigger as it the header

File: README.md [hunk 4]
```diff
@@ -3,3 +3,29 @@
...
+ 
+ ## file structure
+ 
+ frontend-files
+  src/Web_ui
+  
+ Backend-files
+ src/Services
+ 
+ ## installation
```
Comment (amkayondo) [inline README.md:29]: Please reference to https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3 for a better Read Me

**Review by amkayondo — CHANGES_REQUESTED** [review]
reference the modifications with  https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3

**Pull request merged**